chapter_preliminaries/pandas.md

```diff
@@ -110,16 +100,19 @@ X, y
 
 ## Summary
 
-* Like many other extension packages in the vast ecosystem of Python, `pandas` can work together with tensors.
-* Imputation and deletion can be used to handle missing data.
+We barely scratched the surface of what is possible, namely how to partition data columns and how to deal with missing variables in their most basic form. Later on, we will pick up more data processing skills in :ref:`sec_kaggle_house`. The relevant part for now is that we can easily convert tensors from Pandas into tensors managed by the framework of our choice. 
```

Comment:
I think we can rephrase the last line a little.
`The relevant part, for now, is that we can easily convert data from Pandas into tensors managed by the framework of our choice.`